compiler/optimizer/VPHandlers.cpp

```diff
@@ -11252,6 +11249,32 @@ TR::Node *constrainSwitch(OMR::ValuePropagation *vp, TR::Node *node)
            node->setCannotOverflow(true);  // Let evaluator know value range of selector is covered by min and max case
          } // ... else if LongConstraint
       }
+   else
+      {
+      // TODO Can cases outside of the constraint range be removed?
+      TR::Node *selector = node->getFirstChild();
+      bool isGlobal;
+      TR::VPConstraint *constraint = vp->getConstraint(selector, isGlobal);
+      bool isInt64 = selector->getType().isInt64();
+      int64_t maxCase = LONG_MIN;
```

Comment:
```maxCase``` and ```minCase``` do not appear to be used in the new code added. Intentional ?

Reply:
Accidental copy and paste.  I will remove them and push the modified commit later today.